airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/DestinationAcceptanceTest.java

```diff
@@ -614,6 +613,73 @@ public void testIncrementalSync() throws Exception {
         defaultSchema);
   }
 
+  @ParameterizedTest
+  @ArgumentsSource(DataArgumentsProvider.class)
+  public void testIncrementalSyncWithNormalizationDropOneColumn(final String messagesFilename, final String catalogFilename)
+      throws Exception {
+    if (!normalizationFromSpec()) {
+      return;
+    }
+
+    final AirbyteCatalog catalog = Jsons.deserialize(MoreResources.readResource(catalogFilename),
+        AirbyteCatalog.class);
+
+    if (!catalog.getStreams().get(0).getName().equals("exchange_rate")) {
```

Comment:
rather than using `@ParameterizedTest`, you can just reference `DataArgumentsProvider.EXCHANGE_RATE_CONFIG` directly

Reply:
@mfsiega-airbyte what about this comment?

Reply:
@davinchia me wanting to but not getting a chance to spend the time to address this comment is why this PR isn't already merged :) it's not a trivial context switch, so I haven't found time.

Reply:
I think this entire if statement can be removed now?

Reply:
I'll send a follow-up with that cleanup.